### DISCUSSION TITLE
How to move status from 'draft' to 'superseded'?

### DISCUSSION THREAD

**Discussion by munntjlx** [thomvaill/log4brains#50] [Q&A]
I was looking for the workflow for moving an architectural decision from 'draft' to superseded'.

**Comment by ckolumbus** [comment]
We simply create the new DR, review it and when it's moved to  "released" state we  update the old DR in parallel (e.g. same checking) with the new state "superseded" and the link to the new one. 

Is this what you are looking for?

**Reply by munntjlx** [reply]
One note, you have to remove the [type here] or the system doesn't recognise the new status. So I just put 'superseded' and the system worked as planned.

**Comment by munntjlx** [comment]
You just have to change the line to a proper phase, such as 'released' then things work as expected.
